src/main/java/me/i509/fabric/bulkyshulkies/client/extension/QuickShulkerHookClient.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2019-2020 i509VCB
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package me.i509.fabric.bulkyshulkies.client.extension;
+
+import me.i509.fabric.bulkyshulkies.client.screen.Generic11x7Screen;
+import me.i509.fabric.bulkyshulkies.client.screen.Generic13x7Screen;
+import me.i509.fabric.bulkyshulkies.client.screen.Generic9x7Screen;
+import me.i509.fabric.bulkyshulkies.client.screen.ScrollableScreen;
+import me.i509.fabric.bulkyshulkies.extension.QuickShulkerHook;
+import net.fabricmc.fabric.api.client.screen.ScreenProviderRegistry;
+import net.kyrptonaught.quickshulker.api.RegisterQuickShulkerClient;
+
+public class QuickShulkerHookClient implements RegisterQuickShulkerClient {
```

Comment:
This would work, but it's not refereed to by anything else.

Reply:
Forgot to push the changes to the fabric.mod.json, added another entrypoint